### ISSUE TITLE
Save usernames on organization members (i.e. ability to have different usernames in different orgs)

### ISSUE THREAD

**Issue by gustavlrsn** [Edgeryders-Participio/plato-core#5]
Implicates that you'll only be able to sign in to keycloak using your email.

**Comment by Powersource** [comment]
I'm fine with this. But wouldn't it be nice to be able to have one common username for all orgs in some way?

**Comment by gustavlrsn** [comment]
Yeah, it would be pretty nice I guess. If we also need the ability to have different usernames per organization, maybe the keycloak username could be your standard username that you would be able to overwrite in the `organization member` collection for that org? But maybe a risk of it getting a bit complicated..

**Comment by gustavlrsn** [comment]
Discourse might be blocking us from having different usernames per organization actually, I can only register one account/username for an email in Discourse.

**Comment by gustavlrsn** [comment]
Might make it cumbersome to keep usernames in sync when an organization has their own Discourse, as someone might have a Discourse username not be available as a Plato username
